bumble/device.py

```diff
@@ -1532,6 +1540,126 @@ async def stop_advertising(self) -> None:
             self.advertising = False
             self.auto_restart_advertising = False
 
+    async def start_extended_advertising(
```

Comment:
Thanks for adding this, it's nice to have a simple API to start using extended advertising.
I would recommend marking this as an "interim/experimental" API, because I think we'll need to update it soon (I started working on a (slightly more complicated) model that will allow an object-based approach that also unifies with the legacy advertising API. Users can use this API for now until the new one is ready.

Reply:
Add a new decorator to handle this kind of situation.